src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -10,6 +10,7 @@ public class MessageConstants {
     public static final String AADHAAR_SHOULD_NOT_EMPTY = "doctor.aadhaarNumber.should.not.be.empty";
```

Comment:
Rename AADHAAR_SHOULD_NOT_EMPTY to AADHAAR_SHOULD_NOT_BE_EMPTY

Reply:
changed to AADHAAR_SHOULD_NOT_BE_EMPTY